src/EventStore.LogV3.Tests/EventStore.LogV3.Tests.csproj

```diff
@@ -1,21 +1,24 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>net5.0</TargetFramework>
+		<TargetFramework>net6.0</TargetFramework>
 		<GenerateAssemblyInfo>true</GenerateAssemblyInfo>
 		<Platforms>x64;ARM64</Platforms>
 	</PropertyGroup>
 	<ItemGroup>
 		<PackageReference Include="GitHubActionsTestLogger" Version="2.0.1" />
-		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.1" />
-		<PackageReference Include="ObjectLayoutInspector" Version="0.1.2" />
-		<PackageReference Include="System.Linq.Async" Version="5.0.0" />
-		<PackageReference Include="xunit" Version="2.4.1" />
-		<PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+		<PackageReference Include="ObjectLayoutInspector" Version="0.1.4" />
+		<PackageReference Include="System.Linq.Async" Version="6.0.1" />
+		<PackageReference Include="System.Net.Http" Version="4.3.4" />
+		<PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
```

Comment:
here as well, it seems that: Newtonsoft.Json, System.Net.Http and System.Text.RegularExpressions are not required

Reply:
resolving, i'm presuming these were added to make the vulnerability check pass too.

Reply:
yep, xunit takes dependencies on vulnerable packages
https://github.com/xunit/xunit/issues/2568 (regexp)
https://github.com/xunit/xunit/issues/2294 (http)

and Microsoft.NET.Test.Sdk is using the vulnerable newtonsoft package https://github.com/microsoft/vstest/issues/3837